cfme/tests/openstack/cloud/test_volume_backup.py

```diff
@@ -72,3 +129,20 @@ def test_create_volume_backup(volume_backup):
 def test_create_volume_incremental_backup(incremental_backup):
     assert incremental_backup.exists
     assert incremental_backup.size == VOLUME_SIZE
+
+
+def test_incr_backup_of_attached_volume_crud(provider, incremental_backup_of_attached_volume):
+    assert incremental_backup_of_attached_volume.exists
+    assert incremental_backup_of_attached_volume.size == VOLUME_SIZE
+
+    backup_name = incremental_backup_of_attached_volume.name
+    collection = provider.appliance.collections.volume_backups.filter({'provider': provider})
+    incremental_backup_of_attached_volume.delete()
+    view = provider.appliance.browser.create_view(navigator.get_class(collection, 'All').VIEW)
```

Comment:
Required:
When we have `object` and `registered destination` against it then, we can go directly with 
`view = navigate_to(collection, "All")` 
If we are on the same page it will avoid redundant navigation and return a view.